hydradx/apps/fees/fees_volume_comp.py

```diff
@@ -0,0 +1,74 @@
+from matplotlib import pyplot as plt
+import sys, os
+import streamlit as st
+
+project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../"))
+sys.path.append(project_root)
+from hydradx.model.indexer_utils import get_fee_history, get_asset_info_by_ids
+
+c1, c2, c3 = st.columns(3)
+
+with c1:
+    asset_id = st.number_input(
+        "Asset ID",
+        min_value=0, max_value=99999999, value=5, step=1, key="asset_id", format="%d",
+    )
+
+with c2:
```

Comment:
A nice feature would be the ability to set this to a negative number and have that interpreted relative to max_block. Then you could easily get, say, the most recent 1000 blocks without having to know what the current block number is.